cirq-web/cirq_ts/README.md

```diff
@@ -0,0 +1,77 @@
+    - Combining aspects of the visualization that need to be added separately.
+
+This `main.ts` file will also need to be added as an entry point in the `webpack.config.js` file in order for your visualization to be bundled accordingly.
+```javascript
+module.exports = {
+  entry: {
+    bloch_sphere: './src/bloch_sphere/main.ts',
+    ...
+    YOUR_VIZ_NAME: './src/YOUR_VIZ_NAME/main.ts',
+  },
+  ...
+};
+```
+You can learn more about Webpack entry points here: [Webpack Entry Points](https://webpack.js.org/concepts/entry-points/)
+
+### Creating visualization bundle files
+
+Following this structure, you will be able to bundle your visualization by running the command `npx webpack --mode production`. This will build the bundled Javascript file(s) within the `dist/` directory, where you can access and reference them in HTML.
+
+### Developing visualizations
+
+There are two main ways to develop visualizations you are creating in Cirq. The first, and recommended way, is to spin up a Webpack development server and view your visualizations in the browser. You can also develop using Jupyter notebook if you want to easily test integration with Python code. 
+
+#### Hot reloading development server (Recommended)
+Using `webpack-dev-server`, we are able to develop and test visualizations in the browser and have changes update as we're writing the code. You can start this server by running `npm run start` in this directory, and view your work on the port specified by Webpack. This method also requires an `index.html` directory placed within the `dist` folder. You can also manually determine where your index file is served from by modifying where `webpack-dev-server` searches for files in the `webpack.config.js` file:
+```javascript
+module.exports = {
+  ...
+  devServer: {
+    static: path.join(__dirname, 'dist'),
+    public: 'localhost:8080',
+  },
+  ...
+};
+```
+Note that the bundled files that `webpack-dev-server` creates live in memory, so you won't be able to see them. However, you can access them like so:
+```html
+<script src="YOUR_VIZ_NAME.bundle.js"></script>
+```
+
+### Developing in Jupyter Notebook
+An alternative to developing using `webpack-dev-server` is to bundle the Typescript and reference the Javascript output. You can spin up a notebook server with `jupyter notebook`, and bundle (while watching for live changes and updating accordingly) with `npx webpack --mode production --watch`. These processes must run simultaneously. This is especially useful for if you want to work with integrating Python code into your visualization. There's an example notebook `example.ipynb` that provides an example on how to do this.
+
+**NOTE:** In order to access the bundled javascript, you need to include the full path to it: `cirq_ts/dist/YOUR_VIZ_NAME.bundle.js`. If you make any changes to the directory structure, take into account that the path may change as well.
```

Comment:
I see this name as a placeholder for now...when the actual PR goes through the big review we will probably be renaming the whole thing. Something to bring up again in Cirq Sync.

Reply:
We need that prefix, because that will be the actual root level package name in the installation folder, e.g. `<python-env>/lib/site-packages/cirq_ts` and `<python-env>/lib/site-packages/cirq_web` - otherwise it might collide with something called `ts` or `web`, and also then the user will have to say `import web` or `import ts` (although the latter won't be necessary).